projects/cds/src/cms-components/merchandising/carousel/merchandising-carousel/merchandising-carousel.component.html

```diff
@@ -0,0 +1,20 @@
+<cx-carousel
```

Comment:
where's the metadata for the carousel level as we discussed before?

Reply:
This will be done in ticket https://jira.hybris.com/browse/COL-164